pkg/cloudevents/transport/http/transport.go

```diff
@@ -175,7 +175,6 @@ func (t *Transport) obsSend(ctx context.Context, event cloudevents.Event) (*clou
 
 		req.Body = ioutil.NopCloser(bytes.NewBuffer(m.Body))
 		req.ContentLength = int64(len(m.Body))
-		req.Close = true
```

Comment:
This was added because of kafka per @matzew's request.

Reply:
Ben, can you do one thing for me:

keep this deleted and change lines 153-156 to this:

```go
if t.Req != nil {
		req.Method = t.Req.Method
		req.URL = t.Req.URL
		req.Close = t.Req.Close
		copyHeaders(t.Req.Header, req.Header)
	}
```

Reply:
Yes, moving that to be configurable by the user makes sense.